lisa/core/tool.py

```diff
@@ -41,7 +40,13 @@ def canInstall(self) -> bool:
 
     @property
     def isInstalledInternal(self) -> bool:
-        result = self.node.execute(f'bash -c "command -v {self.command}"')
+        if self.node.isLinux:
+            where_command = "command -v"
+        else:
+            where_command = "where"
+        result = self.node.execute(
+            f"{where_command} {self.command}", useBash=True, noInfoLog=True
```

Comment:
Especially when the node may not be Linux, the `useBash` here is confusing.

Reply:
The standard Python lib do same thing. And useBash is necessary on windows internal commands like cd, echo. They are in cmd.exe, not a real executable.

Reply:
I will change it to shell, like Python lib.